.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+files: "nilearn/decomposition"
```

Comment:
Adding new modules and other files will have to be done by modifying this line.
It should be a valid python regex.